mdoc/Test/en.expected-javascript/Mono.DocTest.Generic/GenericBase`1.xml

```diff
@@ -0,0 +1,149 @@
+<Type Name="GenericBase&lt;U&gt;" FullName="Mono.DocTest.Generic.GenericBase&lt;U&gt;">
+  <TypeSignature Language="C#" Value="public class GenericBase&lt;U&gt;" />
+  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit GenericBase`1&lt;U&gt; extends System.Object" />
+  <AssemblyInfo>
+    <AssemblyName>DocTest</AssemblyName>
+    <AssemblyVersion>0.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <TypeParameters>
+    <TypeParameter Name="U" />
+  </TypeParameters>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <typeparam name="U">To be added.</typeparam>
+    <summary>To be added.</summary>
+    <remarks>To be added.</remarks>
+  </Docs>
+  <Members>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public GenericBase ();" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
+      <MemberSignature Language="JavaScript" Value="function GenericBase()" Usage="var genericBase = new GenericBase();" />
```

Comment:
Isn't it not possible to instantiate this class, since it requires a generic type parameter? in this case, `IsSupported` should return false for this constructor ... right?

Reply:
Actually, we can't compile WinRT type if it has generic arguments or any public members with generic arguments. So, if the type isn't supported by WinRT we can treat all its members as unsupported. Will it be a good solution?

Reply:
Ahh, good point ... so in that case, one would never be using, say, `List<T>` directly in javascript. Yes, I think this is a fair solution